tests/foreman/ui/test_user.py

```diff
@@ -913,25 +927,6 @@ def test_negative_delete_user(self):
             self.user.delete(user_name, really=False)
 
     @stubbed()
-    @tier2
-    def test_negative_delete_last_admin(self):
```

Comment:
Thought about that scenario and it didn't seem as a thing that can be automated. With new cleanup approach it becomes even more impossible to do that. Basically, test cases like this should be left to manual QE procedures or performed on automation startup phase. Vote to remove it at all as workarounds seems as too expansive